web/impact/templates/emails/reserve_office_hour_email_to_finalist.html

```diff
@@ -23,6 +23,12 @@
       Yahoo
     </a>
   </li>
+  <li>
```

Comment:
This work is not part of def of done

Reply:
Looks like some sort of merge damage. Removing